pkg/reconciler/delivery/delivery.go

```diff
@@ -143,7 +144,12 @@ func (c *Reconciler) fetchRevisions(cfg *v1.Configuration) (map[string]*v1.Revis
 func (c *Reconciler) fetchPolicy(cfg *v1.Configuration) (*Policy, error) {
 	// there's no need for defensive map query check, because it would have been taken care of in ReconcileKind
 	policyName := cfg.Annotations[delivery.PolicyNameKey]
-	p, err := c.policyLister.Policies(cfg.Namespace).Get(policyName)
+	policyNamespace := cfg.Namespace
+	if s := strings.SplitN(policyName, "/", 2); len(s) > 1 {
```

Comment:
test

Reply:
It's due to a gofmt error, I've fixed it

Reply:
This is new functionality, there should be a unit test that validates this functionality.

Reply:
Added, PTAL. `fetchPolicy` itself is hard to unit test, so I extracted the bits where policy namespace and name are extracted and tested that logic alone.